### PR TITLE
Merge original README

### LINKED ISSUE #7: README merge
After merging the history, it would be interesting to merge also a bit the old README into new one:

# nuclei
ensdf viewer &amp; editor

Note: this repository has been imported in Apr 2021 from https://sourceforge.net/projects/nuclei/ in order to adapt it to Qt5. In 2023 it was merged with the divergent one at https://github.com/martukas/nuclei

Nuclei is a software tool for the displaying of nuclear decay schemes and estimated energy spectra, the calculation of angular γ emission anisotropies, and the automated search for appropriate decay cascade properties. It uses the Evaluated Nuclear Structure Data Files (ENSDF).

A description of Nuclei's functionality and results obtained using its search method was published in Nuclear Instruments and Methods in Physics Research, Section A:

M. Nagl, et al., NIM A 726 (2013), 17-30
[doi:10.1016/j.nima.2013.05.045]

-------------------------
© 2012-2013, Matthias Nagl, II. Physikalisches Institut, Georg-August-Universität Göttingen
http://physik2.uni-goettingen.de/research/2_hofs
-------------------------

# Features

- Exports decay schemes as pdf or svg file
- Allows to select decay cascades for TDPAC spectroscopy
- Computes angular gamma emission anisotropy values (e.g. for TDPAC)
- Shows estimated energy spectra for easier spectrometer calibration
- Powerful decay cascade search functionality

# Prerequisites (Ubuntu 19 - Qt5)

Quazip:
sudo apt install libquazip-dev libgl1-mesa-dev

Libakk:

- cd /opt
- sudo git clone https://github.com/ferdymercury/libakk
- sudo chown $USER -R libakk
- cd libakk
- git checkout dev
- Open with QtCreator and compile
- On QtCreator build directory path, create symlink called libakk pointing to build-...

QtFtp:

- cd /opt
- sudo git clone https://github.com/qt/qtftp
- sudo chown $USER -R qtftp
- cd qtftp
- <QTDIR>/bin/syncqt.pl -version X.Y.Z
- Open with QtCreator and compile
- On QtCreator build directory path, create symlink called qtftp pointing to build-...

Qwt:
- cd /opt
- sudo git clone https://github.com/opencor/qwt
- sudo chown $USER -R qwt
- Open with QtCreator and compile
- cd to build directory generated by QtCreator
- make
- sudo make install

Qxt:
- cd /opt
- sudo git clone https://github.com/ferdymercury/libqxt
- sudo chown $USER -R libqxt
- cd libqxt
- git checkout dev
- ./configure -qmake-bin <QT_DIR>/bin/qmake
- make
- sudo make install
- If some error is found, you might need for nuclei to compile to do this:
- cd /usr/local/Qxt/include
- sudo ln -s QxtCore/qxtglobal.h qxtglobal.h
- sudo ln -s QxtCore/qxtnamespace.h qxtnamespace.h

# Compilation (Ubuntu 19 - Qt5)

- cd /opt
- git clone https://github.com/ferdymercury/nuclei
- sudo chown $USER -R nuclei
- cd nuclei
- git checkout dev
- Open with QtCreator and compile
- Click on Run

# ENSDF database
- For the moment the automatic download of the ENSDF database via FTP is broken
- Instead, download it from https://www.nndc.bnl.gov/ensarchivals/
- Extract it into the same folder e.g. on /opt/
- When running, select load from local copy

### PR DESCRIPTION
**Pull request by ferdymercury**
Fixes https://github.com/martukas/nuclei/issues/7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,21 @@
...
 An Evaluated Nuclear Structure Data
-([ENSDF](https://www.nndc.bnl.gov/ensdf/))
+([ENSDF](https://www.nndc.bnl.gov/ensdf/)) 
 parser, viewer and editor.
 
 A future library for native ENSDF access and manipulation in C++.
 
+## Features
+
+Nuclei is a software tool for the displaying of nuclear decay schemes and estimated energy spectra, the calculation of angular γ emission anisotropies, and the automated search for appropriate decay cascade properties.
```
Comment (martukas) [inline README.md]: I am uncertain if that's still true. In my modernization, I think I removed the sprectra generation feature.

File: README.md [hunk 2]
```diff
@@ -39,6 +46,11 @@ conan remote add conancenter https://center.conan.io
 conan remote add ess-dmsc https://artifactory.esss.lu.se/artifactory/api/conan/ecdc-conan-release
 ```
 
+If on Ubuntu 22, install:
+```bash
+sudo apt install libqt5svg5-dev
```
Comment (martukas) [inline README.md]: I might even simplify this `Building` section with simply a pointer to the circleci script to see what packages are installed. What bulids on CI is the closest thing to a guarantee of functionality. Everything else is hard to state with any certainty and/or maintain.

File: README.md [hunk 3]
```diff
@@ -62,12 +74,21 @@ Parsing errors will be printed in terminal as the files are loaded.
...
 [similar project](https://sourceforge.net/projects/nuclei/)
-by M.A.Nagel.
+by M.A.Nagl, as it can be seen in the git history.
+
+A description of the original Nuclei's functionality and results obtained using its search method was published in Nuclear Instruments and Methods in Physics Research, Section A:
+
+M. Nagl, et al., NIM A 726 (2013), 17-30, [doi:10.1016/j.nima.2013.05.045](https://doi.org/10.1016/j.nima.2013.05.045)
+
+-------------------------
+© 2012-2013, Matthias Nagl, II. Physikalisches Institut, Georg-August-Universität Göttingen
```
Comment (martukas) [inline README.md]: Probably the old license no longer applies? Also, the specific link is dead and just goes to the institute's main page. Maybe we don't need this snippet?

**Review by martukas — CHANGES_REQUESTED** [review]
Thanks! Sorry it took a while to respond to this.
I have provided a few suggestions.

**Review by martukas — APPROVED** [review]
LGTM

(other than CI failing)

**Comment by ferdymercury** [comment]
@php1ic it seems that the circleci does not run well. Could it maybe be a multithreading issue?

**Comment by php1ic** [comment]
Yeah, I have seen this before on circleci, sometimes the build fails if you let things build without specifying how many cores/threads to use.

https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-build-j

Simplest way is to add an argument after `--parallel` on [line 27](https://github.com/martukas/nuclei/blob/main/.circleci/config.yml#L27) of the config file. Looks like it uses a 2 core CPU so that is the number we should use.

**Pull request merged**